device/core/devdoc/device_client_requirements.md

```diff
@@ -192,11 +192,9 @@ The `sendEventBatch` method sends a list of event messages to the IoT Hub as the
 
 #### getTwin(done)
```

Comment:
>getTwin [](start = 5, length = 7)

Is there a requirement for getTwin to connect the transport?

Reply:
it's the transport responsibility to connect (or not if it's already connected). the client doesn't know about the state of the transport connection (and should not)

---
In reply to: [171945976](https://github.com/pierreca/azure-iot-sdk-node/pull/2#discussion_r171945976) [](ancestors = 171945976)